src/checkPassword.test.js

```diff
@@ -7,13 +7,52 @@ describe(`Function 'checkPassword':`, () => {
     expect(checkPassword).toBeInstanceOf(Function);
   });
 
+  it(`Should accept a string value`, () => {
+    const password = 'Password1&';
```

Comment:
This test case is unnecessary, as it's just checking the input type, which is not the purpose of the checkPassword function. Please remove this test case.